Content.Server/Physics/Components/RandomWalkComponent.cs

```diff
@@ -29,6 +30,18 @@ public sealed class RandomWalkComponent : Component
     [ViewVariables(VVAccess.ReadWrite)]
     public float AccumulatorRatio = 0.0f;
 
+    /// <summary>
+    /// The vector by which the random walk direction is biased.
+    /// </summary>
+    [DataField("BiasVector")]
+    [ViewVariables(VVAccess.ReadWrite)]
+    public Vector2 BiasVector = new Vector2(0f, 0f);
+    
+    /// <summary>
+    /// Whether to set BiasVector to (0, 0) every random walk update.
+    /// </summary>
+    public bool ResetBiasOnWalk = true;
```

Comment:
DataField.

Reply:
done, added read-write VV too since you could want to change it